File: Example/RecordLife/Classes/View/XJYViewController.m
//
//  XJYViewController.m
//  RecordLife
//
//  Created by 谢俊逸 on 17/01/2017.
//  Copyright © 2017 谢俊逸. All rights reserved.
//

#import "XJYViewController.h"

@interface XJYViewController ()

@end

@implementation XJYViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    self.navigationController.navigationBar.tintColor = [UIColor blackColor];
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


// this is Apple's recommended place for adding/updating constraints
- (void)updateViewConstraints {
    [super updateViewConstraints];
}

// tell UIKit that you are using AutoLayout
+ (BOOL)requiresConstraintBasedLayout {
    return YES;
}
@end

File: XJYChartDemo/XJYChartDemo-Swift/View/XJYViewController.swift
//
//  XJYViewController.swift
//  XJYChartDemo-Swift
//
//  Created by Kelly Roach on 8/17/18.
//  Copyright © 2018 JunyiXie. All rights reserved.
//

import UIKit

public class XJYViewController : UIViewController {
    // tell UIKit that you are using AutoLayout
    class var requiresConstraintBasedLayout: Bool {
        return true
    }

    override public func viewDidLoad() {
        super.viewDidLoad()
        navigationController?.navigationBar.tintColor = UIColor.black
    }

    override public func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }

    // this is Apple's recommended place for adding/updating constraints
    override public func updateViewConstraints() {
        super.updateViewConstraints()
    }
}
